cms/settings.py

```
@@ -10,6 +10,7 @@
 ALLOWED_HOSTS = [
     "127.0.0.1",
     "localhost",
+    "0.0.0.0",
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)